src/App.jsx

```diff
@@ -25,21 +26,12 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Olya</h2>
-      <p className="Person__partner">Maksym is my husband</p>
-    </section>
-
-    <section className="Person">
-      <h2 className="Person__name">My name is Alex</h2>
-      <p className="Person__age">I am 25</p>
-      <p className="Person__partner">I am not married</p>
-    </section>
+
+    <Person person={misha} />
+
+    <Person person={olya} />
+
+    <Person person={alex} />
+
   </div>
```

Comment:
```suggestion
    <Person person={misha} />

    <Person person={olya} />

    <Person person={alex} />
  </div>
```
Don't leave empty lines between parent and child elements